solana-sample/src/pb.rs

```diff
@@ -0,0 +1,19 @@
+#[rustfmt::skip]
+#[path = "../target/pb/messari.common.v1.rs"]
+pub(in crate::pb) mod common_v1;
```

Comment:
proto files aren't being used so we could remove them from the substream for now?